src/systemctl/systemctl-compat-shutdown.c

```diff
@@ -129,8 +130,12 @@ int shutdown_parse_argv(int argc, char *argv[]) {
                         log_error("Failed to parse time specification: %s", argv[optind]);
                         return r;
                 }
-        } else
-                arg_when = now(CLOCK_REALTIME) + USEC_PER_MINUTE;
+        } else {
+                if (logind_has_maintenance_window())
+                        arg_when = USEC_INFINITY; /* logind chooses on server side */
+                else
+                        arg_when = now(CLOCK_REALTIME) + USEC_PER_MINUTE;
+        }
```

Comment:
please do not extend sysv shutdown, it's a compat iface

Reply:
leaving in as discussed above